mypy.ini

```diff
@@ -1,6 +1,5 @@
 [mypy]
 python_version = 3.11
-plugins = pydantic.mypy
```

Comment:
Why are we removing this?

Reply:
Because the plugin was crashing, I forgot to actually find a solution.

Bumping to pydantic 1.10.9 Instead of 1.10.2 fixed it.